overlord/devicestate/handlers_install.go

```diff
@@ -447,6 +492,22 @@ func (m *DeviceManager) doRestartSystemToRunMode(t *state.Task, _ *tomb.Tomb) er
 		return err
 	}
 
+	// write timing information, the lock must be unset for this
```

Comment:
```suggestion
	// write timing information, the state must be inlocked
```
also should probably be placed right before writeTimings()